Clarify NPoS Election Algorithms on Polkadot (#3466)
w3f/polkadot-wiki: docs/learn/learn-phragmen.md

---
id: learn-phragmen
title: NPoS Election Algorithms
sidebar_label: NPoS Election Algorithms
description: Learn about the election method used on Polkadot's Nominated Proof of Staking.
keywords: [phragmen, sequential phragmén method, elections, algorithm, phragmms]
slug: ../learn-phragmen
---

## NPoS Election Algorithms

Since validators are paid almost equally in Polkadot in each era, it is important that the stake behind each validator 

> is uniformaly spread out. Any algorithm for Nominated Proof of Staking (NPoS) on Polkadot should try to optimize three 
> metrics when computing an election solution graph with nominators and validators:

Reviewer: ```suggestion
is uniformly spread out. An election algorithm for Nominated Proof of Staking (NPoS) on Polkadot will try to optimize three 
metrics when computing a solution graph of nominators and validators:
```
